src/screens/ProfileScreen/elements/AnonymousTab.js

```diff
@@ -23,24 +23,20 @@ const styles = StyleSheet.create({
     fontSize: 14,
     lineHeight: 17,
     textAlign: 'center',
-    color: isActive ? colors.bondi_blue : colors.blackgrey
+    color: isActive ? COLORS.bondi_blue : COLORS.blackgrey
   }),
   anonymousPostTextSubtitle: (isActive) => ({
     fontFamily: isActive ? fonts.inter[600] : fonts.inter[400],
     fontSize: 10,
     textAlign: 'center',
-    color: isActive ? colors.bondi_blue : colors.blackgrey
+    color: isActive ? COLORS.holyTosca : COLORS.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color changes in these lines are consistent with the new `COLORS` import. It's important to ensure that the new color values (`COLORS.bondi_blue`, `COLORS.holyTosca`, `COLORS.blackgrey`) are defined in the `COLORS` object in the `utils/theme` module.
```diff
-    color: isActive ? colors.bondi_blue : colors.blackgrey
+    color: isActive ? COLORS.bondi_blue : COLORS.blackgrey

-    color: isActive ? colors.bondi_blue : colors.blackgrey
+    color: isActive ? COLORS.holyTosca : COLORS.blackgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->